pip-purge

```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env python
 
 import envoy
+import sys
 
 def main():
+    if hasattr(sys, "real_prefix"):
```

Comment:
This could probably be a return early, instead of throwing it all into an conditional. Something like this.
 `def main():`

```python
if not hasattr(sys, 'real_path'):
    print('Warning! You are not in a virtualenv. This will purge all system-level packages.')
    sys.exit(1)

... rest of code here

```